### PR TITLE
Rd 2548 add scaling example

### PR DESCRIPTION
**Pull request by AdarShaked**

### CODE REVIEW

File: scaling-example/resources/ansible/uninstall-playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+- hosts: all
+  connection: local
+  tasks:
+
+#    - name: Terminate vm
```
Comment (EarthmanT) [inline scaling-example/resources/ansible/uninstall-playbook.yaml:5]: Do we need to keep commented values?

File: scaling-example/ansible.yaml [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    type: cloudify.policies.scaling
+    properties:
+      default_instances: 1
+    targets: [vm-group]
+
+capabilities:
+
+#  endpoint:
+#    description: The external endpoint of the application.
+#    value:  { get_attribute: [aws_infra_info, public_ip] }
```
Comment (EarthmanT) [inline scaling-example/ansible.yaml:208]: Keeping? Not keeping?

**Review by EarthmanT — COMMENTED** [review]
Commented values comments applies to all files.
Also, can we get a readme?

File: scaling-example/resources/ansible/uninstall-playbook.yaml (hunk 1 above)
Comment (AdarShaked) [reply scaling-example/resources/ansible/uninstall-playbook.yaml:5]: no

File: scaling-example/ansible.yaml (hunk 2 above)
Comment (AdarShaked) [reply scaling-example/ansible.yaml:208]: no

File: scaling-example/resources/ansible/create_eni_and_vm_playbook.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      set_fact:
+        output_dict:
+          public_ip:  "{{ public_ip }}"
+          ec2_instance_ids: "{{ ec2.instance_ids}}"
+          eni_id: "{{ eni_id }}"
+
+    - name: Write variables to infra info file
+      copy:
+        content: "{{ output_dict | to_nice_json }}"
+        dest: "{{ infra_info_file }}"
```
Comment (EarthmanT) [inline scaling-example/resources/ansible/create_eni_and_vm_playbook.yaml:74]:
```suggestion
        dest: "{{ infra_info_file }}"
        
```

File: scaling-example/resources/ansible/delete_eni_and_vm_playbook.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+    - name: Delete elastic network interface
+      ec2_eni:
+        eni_id: "{{ eni_id }}"
+        state: absent
+
+    - name: Release elastic IP
+      ec2_eip:
+        public_ip: "{{ public_ip }}"
+        state: absent
```
Comment (EarthmanT) [inline scaling-example/resources/ansible/delete_eni_and_vm_playbook.yaml:19]:
```suggestion
        state: absent
        
```

File: scaling-example/resources/ansible/inventory.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+all:
+  hosts:
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: "{{ansible_playbook_python}}"
```
Comment (EarthmanT) [inline scaling-example/resources/ansible/inventory.yaml:5]:
```suggestion
      ansible_python_interpreter: "{{ansible_playbook_python}}"

```

File: scaling-example/resources/ansible/playbook.yaml [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+          vpc_name: "{{ vpc_name }}"
+          subnet_id: "{{ vpc_subnet_id }}"
+          security_group_id: "{{ security_group_id }}"
+          internet_gateway_id: "{{internet_gateway_id}}"
+          route_table_id: "{{route_table_id}}"
+
+    - name: Write variables to infra info file
+      copy:
+        content: "{{ output_dict | to_nice_json }}"
+        dest: "{{ infra_info_file }}"
```
Comment (EarthmanT) [inline scaling-example/resources/ansible/playbook.yaml:123]:
```suggestion
        dest: "{{ infra_info_file }}"

```

File: scaling-example/ansible.yaml [hunk 7]
```diff
@@ -0,0 +1,213 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This blueprint creates an AWS infrastructure environment using Ansible.
+  This is an example of blueprint that supports scale workflow.
+  libselinux-python/libselinux-python3 on the manager VM is prerequisite.
+
+imports:
+  - https://cloudify.co/spec/cloudify/6.0.0/types.yaml
+  - plugin:cloudify-ansible-plugin
```
Comment (EarthmanT) [inline scaling-example/ansible.yaml:10]: Should we constrain Ansible?

File: .cicd/supported_examples.json [hunk 8]
```diff
@@ -172,5 +172,8 @@
   "utilities-examples/cloudify_terminal": [
     "linux-ssh.yaml",
     "fortigate.yaml"
+  ],
+    "scaling-example": [
+    "ansible.yaml"
```
Comment (EarthmanT) [inline .cicd/supported_examples.json:177]: I don't really want to add this to testing yet, until we have all the other infras.

File: scaling-example/ansible.yaml (hunk 7 above)
Comment (AdarShaked) [reply scaling-example/ansible.yaml:10]: Not really, I'm not using your new features only the run operation.

File: .cicd/supported_examples.json (hunk 8 above)
Comment (AdarShaked) [reply .cicd/supported_examples.json:177]: It is just the blueprint validation.

**Review by EarthmanT — APPROVED** [review]

**Pull request merged**